src/org/usfirst/frc4904/robot/commands/Shoot.java

```diff
@@ -0,0 +1,22 @@
+package org.usfirst.frc4904.robot.commands;
+
+
+import org.usfirst.frc4904.robot.RobotMap;
+import org.usfirst.frc4904.robot.subsystems.Shooter;
+import org.usfirst.frc4904.standard.commands.RunFor;
+import edu.wpi.first.wpilibj.command.Command;
+import edu.wpi.first.wpilibj.command.CommandGroup;
+
+public class Shoot extends CommandGroup {
```

Comment:
Is there a reason this command does not automatically bring the flywheel up to speed?